src/App.js

```diff
@@ -1,5 +1,74 @@
+const { Console } = require('@woowacourse/mission-utils');
+const InputView = require('./views/InputView');
+const OutputView = require('./views/OutputView');
+const BridgeMaker = require('./BridgeMaker');
+const BridgeRandomNumberGenerator = require('./BridgeRandomNumberGenerator');
+const Bridge = require('./Bridge');
+const BridgeGame = require('./BridgeGame');
+const { MOVE_RESULT } = require('./utils/constants');
+
 class App {
-  play() {}
+  constructor() {
+    this.game = null;
+  }
+
```

Comment:
`game` 속성이 `App`이 생성될 때가 아닌 `start`method에서 초기화된다면 생성자 내부가 아니라 필드에 넣는 것은 어떨까요? 😃

```suggestion
  game;

```